src/main.js

```diff
@@ -55,6 +57,12 @@ function showMainPage() {
   savedPostersSection.classList.add('hidden');
 }
 
+function displayMainPage() {
+  mainPoster.classList.remove('hidden');
+  posterForm.classList.remove('hidden');
+}
+
+
```

Comment:
Hopefully we can find a way to turn all of these display functions into one function we can call for each button.